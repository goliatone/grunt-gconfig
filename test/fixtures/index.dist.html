<!DOCTYPE html>
<html>
<head>
	<title>Test GConfig grunt task: Dist Environment</title>
</head>
<body>
</body>
</html>