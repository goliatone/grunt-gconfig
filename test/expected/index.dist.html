<!DOCTYPE html>
<html>
<head>
	<title>Test GConfig grunt task: Dist Environment</title>
	<!--gconfig-->
    <meta name="platform:ip" content="10.0.2.135">
    <meta name="platform:port" content="9000">
    <meta name="platform:endpoint" content="ws/cs/endpoint">
    <meta name="platform:clientid" content="wits">
    <meta name="platform:useridid" content="goliatone">
    <meta name="app:endpoint" content="ws://{{ip}}:{{port}}/{{endpoint}}/{{clientid}}/{{exhibitid}}">
    <!--@gconfig-->
</head>
<body>
</body>
</html>